test/cases/ostree-ami-image.sh

```diff
@@ -690,12 +777,22 @@ for _ in $(seq 0 9); do
 done
 cat "${INSTANCE_OUT_INFO}"
 
+# Check instance has been deployed correctly
+check_result
+
+INSTANCE_ID=$(jq -r '.Instances[].InstanceId' "${INSTANCE_OUT_INFO}")
+
 # wait for instance running
-sleep 5
+aws ec2 wait instance-running \
+    --instance-ids "$INSTANCE_ID"
```

Comment:
It took me a while to find the functional change here in this PR.  Can we split this into multiple commits so that the formatting changes are in one and the main functional change is in another?

Reply:
Yes, you're right. Fixed. Thanks.